packages/server/lib/project-base.ts

```diff
@@ -10,7 +10,7 @@ import { ServerCt, SocketCt } from '@packages/server-ct'
 import { SocketE2E } from './socket-e2e'
```

Comment:
I used `git mv` on the above file, so although it looks like a deletion, the git history should be maintained.

Reply:
@lmiller1990 I think that's only the case if it doesn't get squash merged

Reply:
We should make sure we do not squash merge JS -> TS migration moving forward.